docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,480 @@
+
+## Create SSH access
+
+In order copy certs between machines, you will need to enable SSH access for `scp`.
+
+1. First, open new new tabs in your shell for `etcd1` and `etcd2`. Ensure you are SSHed into all three machines and then run the following (it will be a lot quicker if you use tmux syncing - to do this in iTerm enter `cmd+shift+i`):
+
+    ```
+    export PEER_NAME=$(hostname)
+    export PUBLIC_IP=$(ip addr show eth0 | grep -Po 'inet \K[\d.]+')
+    export PRIVATE_IP=$(ip addr show eth1 | grep -Po 'inet \K[\d.]+')
+    ```
+
+1. Next, generate some SSH keys for the boxes:
+
+    ```
+    ssh-keygen -t rsa -b 4096 -C "<email>"
+    ```
+
+    Make sure to replace `<email>` with your email, a placeholder, or an empty string. Keep hitting enter until files exist in `~/.ssh`.
+
+1. Output the contents of the public key file for `etcd1` and `etcd2`, like so:
+
+    ```
+    cat ~/.ssh/id_rsa.pub
+    ```
+
+1. Finally, copy the output for each and paste them into `etcd0`'s `~/.ssh/authorized_keys` file. This will permit `etcd1` and `etcd2` to SSH in to the machine.
+
+## Generate etcd server and peer certs
+
+1. In order to generate certs, each etcd machine needs the root CA generated by `etcd0`. On `etcd1` and `etcd2`, run the following:
+
+    ```
+    mkdir -p /etc/kubernetes/pki/etcd
+    cd /etc/kubernetes/pki/etcd
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca.pem .
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/ca-key.pem .
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client.pem .
+    scp root@<etcd0-ip-address>:/etc/kubernetes/pki/etcd/client-key.pem .
+    ```
+
+    Where `<etcd0-ip-address>` corresponds to the public or private IPv4 of `etcd0`.
+
+1. Once this is done, run the following on all etcd machines:
+
+    ```
+    cfssl print-defaults csr > config.json
+    sed -i '0,/CN/{s/example\.net/'"$PEER_NAME"'/}' config.json
+    sed -i 's/www\.example\.net/'"$PRIVATE_IP"'/' config.json
+    sed -i 's/example\.net/'"$PUBLIC_IP"'/' config.json
+
+    cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=server config.json | cfssljson -bare server
+    cfssl gencert -ca=ca.pem -ca-key=ca-key.pem -config=ca-config.json -profile=peer config.json | cfssljson -bare peer
+    ```
+
+    The above will replace the default configuration with your machine's hostname as the peer name, and its IP addresses. Make sure
+    these are correct before generating the certs. If you found an error, reconfigure `config.json` and re-run the `cfssl` commands.
+
+This will result in the following files: `peer.pem`, `peer-key.pem`, `server.pem`, `server-key.pem`.
+
+## Run etcd
+
+Now that all the certificates have been generated, you will now install and set up etcd on each machine. 
+
+{% capture choose %}
+Please select one of the tabs to see installation instructions for the respective way to run etcd.
+{% endcapture %}
+
+{% capture systemd %}
+
+1. First install etcd binaries like so:
```

Comment:
make a note that 3.1.10 is the preferred version for v1.9, for other k8s versions please consult the changelog